client/app/models/step.coffee

```diff
@@ -0,0 +1,17 @@
+Backbone = require 'backbone'
+
+# StepModel
+# Backbone wrapper for Onboarding's Step models
+# This Class is a proxy between Backone's logic and native Onboarding Steps
+module.exports = class StepModel extends Backbone.Model
+
+    # Map needed property to current model
+    initialize: ({step}) ->
```

Comment:
I think if the ```Step``` class is an exclusive class of ```Onboarding``` we can keep it like that. But if this should be an extern class that should be renamed.